mypy/checkstrformat.py

```diff
@@ -163,6 +165,32 @@ def has_star(self) -> bool:
         return self.width == '*' or self.precision == '*'
 
 
+class LineNoChanger(TraverserVisitor):
```

Comment:
nit: can we name this LineNumChanger (or make the n lower case?) sounds like it's not changing the lines :P

Reply:
Done.